backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -43,14 +43,45 @@
   }
 }
 
+// Page Layout
+//---------------------------------------------------
+
+html {
+  height: 100%;
+}
+
+body {
+  position: relative;
+  min-height: 100%;
+}
+
+.admin-nav {
+  @include position(fixed, 0 null 0 0);
```

Comment:
With a fixed position we get problems with smaller screens (like my 13" MacBook Pro) and large sub navigations, like the products tab:

![monosnap 2015-11-20 23-22-19](https://cloud.githubusercontent.com/assets/42868/11313579/32012f2a-8fde-11e5-80a5-0ce312db664b.png)

We even can't make the sidebar scrollable, since the flyout would overflow and cause horizontal scrollbars. So, we need to stick with absolute positioning. 

I'd really love to have sticky main navigation, but the downsides are too large, I guess.